modules/openxr/openxr_interface.cpp

```diff
@@ -1198,6 +1196,11 @@ Array OpenXRInterface::get_supported_environment_blend_modes() {
 				WARN_PRINT("Unsupported blend mode found: " + String::num_int64(int64_t(env_blend_modes[i])));
 		}
 	}
+
+	if (openxr_api->is_emulating_environment_blend_mode_alpha_blend() && modes.find(XR_ENV_BLEND_MODE_ALPHA_BLEND) == -1) {
```

Comment:
Ah, you added it here... I think it should be moved into `openxr_api`, but its a bit of a nitpicking thing. The hope is that eventually vendors like Meta, Pico and HTC jump onboard and actually embed this logic on the runtime side, and we'll be able to remove the logic we're adding here from `openxr_api`.

From my discussions I do get the impression that this was always the intent, but we may have gone down too far the rabbit hole to turn back.